modules/installing-ocp-agent-tui.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-tui_{context}"]
+= Verifying that the current installation host can pull release images
+
+After you boot the agent image and network services are made available to the host, the agent console application performs a pull check to verify that the current host can retrieve release images.
+
+If the primary pull check passes, the application can be quit and the installation will proceed. If the pull check fails, the application will perform additional checks (as seen in the `Additional checks` section of the TUI) to help the user troubleshoot the problem. A failure for any of the additional checks is not necessarily critical as long as the primary pull check succeeds.
+
+If there are host network configuration issues that may cause an installation to fail, you can use the console application to make adjustments to your network configurations.
+
+[IMPORTANT]
+====
+If the agent console application detects host network configuration issues, the installation workflow will be halted until the user manually stops the console application and signals the intention to proceed.
+====
+
+.Procedure
+
+. Wait for the agent console application to check whether or not the configured release image can be pulled from a registry.
+
+. If the agent console application states that the installer connectivity checks have passed, either wait for the prompt to time out or select *No*. The application closes and the installation proceeds.
++
+[NOTE]
+====
+You may still choose to view or change network configuration settings even if the connectivity checks have passed.
+
+However, if you choose to interact with the agent console application rather than letting it time out, you must manually quit the TUI to proceed with the installation.
+====
+
+. If the agent console application checks have failed (which is indicated by a red icon beside the `Release image URL` pull check), use the following steps to reconfigure the host's network settings:
+
+.. Read the `Check Errors` section of the TUI.
+This section displays error messages specific to the failed checks.
++
+image::agent-tui-home.png[The home screen of the agent console application  displaying check errors, indicating a failed check]
+
```

Comment:
General comment for this PR. I believe instances of  `select` should be `click`. [1]

[1] https://www.ibm.com/docs/en/ibm-style?topic=elements-mouse-buttons

Reply:
I agree with the rule but in this case, the TUI/console application is something that you can only navigate with a keyboard (at least as of now). So in this case, a user can only make selections by navigating with arrow keys and pressing enter on their keyboard. Is "select" an appropriate option in that context?

Reply:
Ah. I see. Yes. I believe so.